carbon-design-system/carbon
# feat(codemods): add codemod for updating icons packages

Closes #8516 

This PR adds in the initial codemod for the Icon API updates. There will be a separate PR that will include all of the component updates. This also introduces the migration guide for `@carbon/icons-react` for v11.

#### Changelog

**New**

- Add new transform in `codemods` for updating icons-react imports

**Changed**

**Removed**

#### Testing / Reviewing

- Go into the codemods directory `cd codemods`
- Run the following command `yarn jscodeshift --parser babylon -t transforms/icons-react-size-prop.js '../packages/react/src/components'`
- Verify that files have changed with the intended API updates
